exercises/practice/queen-attack/.meta/test_template.tera

```diff
@@ -1,4 +1,4 @@
-use queen_attack::{ChessPiece, ChessPosition, Queen};
+use queen_attack::*;
```

Comment:
This was inconsistent, must've been a mistake of mine. The change doesn't have any effect though.